ReplayKitExample/ReplayKitExample/ReplayKitVideoSource.swift

```diff
@@ -83,6 +87,16 @@ class ReplayKitVideoSource: NSObject, TVIVideoCapturer {
     }
 
     func stopCapture() {
+        if let yBuffer = downscaleYPlaneBuffer {
+            free(yBuffer)
```

Comment:
reset `yBuffer = nil;` after freeing it.

Reply:
Should be okay since yBuffer goes out of scope.

Reply:
As discussed lets move this logic to `deinit`